{"name": "test_login", "status": "broken", "statusDetails": {"message": "AttributeError: 'async_generator' object has no attribute 'goto'", "trace": "page = <async_generator object page at 0x000001F7CE30A740>\nenv_vars = {'password': 'User@123', 'url': 'https://worldamenities.com/', 'username': 'user123@gmail.com'}\n\n    @pytest.mark.smoke\n    @pytest.mark.asyncio\n    async def test_login(page, env_vars):\n        url = env_vars['url']\n        username = env_vars['username']\n        password = env_vars['password']\n    \n        login_page = LoginPage(page)\n>       await login_page.goto(url)\n\ntests\\test_WAWorld.py:13: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.LoginPage.LoginPage object at 0x000001F7CD545790>, url = 'https://worldamenities.com/'\n\n    async def goto(self, url: str):\n>       await self.page.goto(url)\nE       AttributeError: 'async_generator' object has no attribute 'goto'\n\npages\\LoginPage.py:28: AttributeError"}, "start": 1720145271091, "stop": 1720145271093, "uuid": "3fd87711-cbcf-41dc-87a0-400212441f0f", "historyId": "8bfe3b445a5dfa2b16b8ebe305212da8", "testCaseId": "8bfe3b445a5dfa2b16b8ebe305212da8", "fullName": "tests.test_WAWorld#test_login", "labels": [{"name": "tag", "value": "asyncio"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_WAWorld"}, {"name": "host", "value": "Feuji-V005L-R"}, {"name": "thread", "value": "21004-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_WAWorld"}]}